### PR TITLE
Add WNP for Firefox 122 [fix #14058]

### LINKED ISSUE #14058: What's New page for Firefox 122
## Description

A what's new page for Firefox 122.0, release on **23 January, 2024**

Project brief with other details and links to copy/design: https://docs.google.com/document/d/1pdAERcVuX8-PPGip4YXYEbLKJTekGsjNiwtp_RrluQk/

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Adds the what's new pages for Firefox 122.0, releasing **23 January, 2023**

en-US and en-CA browsers get a page promoting privacy features, with a CTA to either set Firefox as the default browser or see other privacy-related products (if Firefox is already their default).

en-GB, fr, de, it, pl, and es-ES browsers get a page promoting Mozilla accounts, which falls back to a Firefox mobile promotion if they're signed into sync (so we know they already have an account).

## Issue / Bugzilla link
#14058 

## Testing
Test in Firefox when set to default and not default.
US English: http://localhost:8000/en-US/firefox/122.0/whatsnew/

Test in Firefox both signed in and signed out.
GB English: http://localhost:8000/en-GB/firefox/122.0/whatsnew/
Spanish: http://localhost:8000/es-ES/firefox/122.0/whatsnew/
German: http://localhost:8000/de/firefox/122.0/whatsnew/
French: http://localhost:8000/fr/firefox/122.0/whatsnew/
Italian: http://localhost:8000/it/firefox/122.0/whatsnew/
Polish: http://localhost:8000/pl/firefox/122.0/whatsnew/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx122-na.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+{% block content %}
+<main class="content-wrapper mzp-t-firefox">
+  {% include 'firefox/whatsnew/includes/header.html' %}
+
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-md">
+      <h1 class="wnp-main-title">Your privacy always comes first</h1>
+      <p class="wnp-main-tagline">And in a private browsing window, your protections go even further.</p>
+
+      <table class="mzp-u-data-table compare-table">
```
Comment (nathan-barrett) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx122-na.html:31]: 🪑

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx122-na.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+          </tr>
+        </tbody>
+        <tfoot>
+          <tr>
+            <td colspan="7" role="presentation" class="compare-table-legend">
+              <span class="legend-label">Legend:</span>
+              <span class="compare-icon compare-icon-yes">Yes</span> Yes
+              <span class="compare-icon compare-icon-some">Yes</span> Somewhat
+              <span class="compare-icon compare-icon-no">Yes</span> No
+            </td>
```
Comment (nathan-barrett) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx122-na.html:106]: You could use a fieldset and legend tags here i think? Not 100% sure on it

**Review by nathan-barrett — APPROVED** [review]
Really nice work - GA and GA4 looks good to me, but maybe @stephaniehobson wants to also take a peek. r+wc 🌮

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx122-na.html (hunk 2 above)
Comment (craigcook) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx122-na.html:106]: I think that's really just for forms, and alas this isn't that kind of legend. 🐉

**Review by stephaniehobson — APPROVED** [review]
R+ 📈

**Pull request merged**